pkg/spec/spec.go

```diff
@@ -553,6 +554,35 @@ func addRlimits(config *CreateConfig, g *generate.Generator) error {
 	)
 
 	for _, u := range config.Resources.Ulimit {
+		if u == "host" {
+			if len(config.Resources.Ulimit) != 1 {
+				return errors.New("ulimit can use host only once")
+			}
+
+			type Rlimit struct {
```

Comment:
Does this need to be public? It's purely scoped within this loop

Reply:
good point.  I've pushed a new version